File: encapsulation.cpp
// Encapsulation is a process of wrapping code and data together into a single unit,
#include <iostream>

using namespace std;

class Rectangle {
  public:
    
    int length;
    int height;

    // Constructor to initialize variables
    Rectangle(int l, int h) 
    {
        length=l;
        height=h;
    } 

    
    int Area() {
      return length * height;
    }
};

int main() {
  
  Rectangle rect(8, 6);

  cout << "Area = " << rect.Area();

  return 0;
}

File: encapsulation.py
# Encapsulation is a process of wrapping code and data together into a single unit,

class Rectangle:
    def __init__(self,length,height):
        self.length = length
        self.height = height
    def Area(self):
      return self.length * self.height
    
rect1 = Rectangle(10,20)
area = rect1.Area()
print(area)